app/main.py

```diff
@@ -1 +1,45 @@
-# write your code here
+from os import (
+    makedirs,
+    path
+)
+
+from shutil import move
+
+
+class MoveCommandSyntaxError(Exception):
+    pass
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+
+    if command[0] != "mv":
+        raise MoveCommandSyntaxError("Move command not found")
+
+    if len(command) != 3:
+        raise MoveCommandSyntaxError(
+            "Invalid move command syntax.\n"
+            "For a successful moving of file "
+            "the command should look like this: "
+            "mv file.txt some_dir/new_file.txt"
+        )
+
+    if path.exists(command[-1]):
```

Comment:
PermissionError u need to use in try except where u read your file, because user can give you a file which u can't read, u don't have permission